docusaurus.config.js

```diff
@@ -256,7 +256,7 @@ export default {
           label: 'FAQ',
         },
         {
-          href: 'https://chromewebstore.google.com/detail/universal-profiles/abpickdkkbnbcoepogfhkhennhfhehfn',
+          href: 'install-up-browser-extension',
```

Comment:
It should be:


```
/install-up-browser-extension
```

Otherwise if you are on page `/tools` the link will be to `tools/install-up-browser-extension` which does not exist, this is why you had build issues.

<img width="1043" alt="image" src="https://github.com/lukso-network/docs/assets/477945/fc999788-ee84-4671-adb1-cbea4e3911f6">